src/OpenLoco/src/Audio/Audio.cpp

```diff
@@ -844,16 +844,13 @@ namespace OpenLoco::Audio
             const auto centre = mainViewport->getCentreMapPosition();
             const auto topLeft = World::toTileSpace(centre) - World::TilePos2{ 5, 5 };
             const auto bottomRight = topLeft + World::TilePos2{ 11, 11 };
-            World::TilePosRangeView searchRange(topLeft, bottomRight);
+            const auto searchRange = World::getClampedRange(topLeft, bottomRight);
```

Comment:
Following notes are just for me to confirm I've looked at the loop:
Vanilla <= 0x2FFF x/y (didn't have a < 0 check)